pycode/memilio-surrogatemodel/memilio/surrogatemodel_test/test_surrogatemodel_ode_secir_simple.py

```diff
@@ -91,19 +92,56 @@ def test_data_generation_save(self):
         self.assertEqual(os.listdir(self.path),
                          ['data_secir_simple.pickle'])
 
+    def test_structures_networks(self):
+
+        model_mlp_multi_input_single_output = network_architectures.mlp_multi_input_single_output()
+        self.assertEqual(len(model_mlp_multi_input_single_output.layers), 5)
+        input_zero = np.zeros((1, 5, 8))
+        output_zeros = model_mlp_multi_input_single_output(input_zero)
+        self.assertEqual(output_zeros.shape[0], 1)
+        self.assertEqual(output_zeros.shape[1], 1)
+        self.assertEqual(output_zeros.shape[2], 8)
+
+        model_lstm_single = network_architectures.lstm_network_multi_input_single_output()
+        self.assertEqual(len(model_lstm_single.layers), 2)
+        input_zero = np.zeros((1, 1, 8))
+        output_zeros = model_lstm_single(input_zero)
+        self.assertEqual(output_zeros.shape[0], 1)
+        self.assertEqual(output_zeros.shape[1], 1)
+        self.assertEqual(output_zeros.shape[2], 8)
+
+        label_width = 5
+        model_cnn = network_architectures.cnn_multi_input_multi_output(
+            label_width)
+        self.assertEqual(len(model_cnn.layers), 4)
+        input_zero = np.zeros((1, 5, 8))
+        output_zeros = model_cnn(input_zero)
+        self.assertEqual(output_zeros.shape[0], 1)
+        self.assertEqual(output_zeros.shape[1], 5)
+        self.assertEqual(output_zeros.shape[2], 8)
+
+        model_lstm_multi = network_architectures.lstm_multi_input_multi_output(
+            label_width)
+        self.assertEqual(len(model_lstm_multi.layers), 3)
+        input_zero = np.zeros((1, 5, 8))
+        output_zeros = model_lstm_multi(input_zero)
+        self.assertEqual(output_zeros.shape[0], 1)
+        self.assertEqual(output_zeros.shape[1], 5)
+        self.assertEqual(output_zeros.shape[2], 8)
+
     def test_network_fit(self):
 
         max_epochs = 5
 
         # models with single output
-        model_single = different_networks.single_output()
-        model_dense_multi_input = different_networks.multilayer_multi_input()
-        model_lstm_single = different_networks.lstm_network_multi_input()
+        model_mlp_multi_input_single_output = network_architectures.mlp_multi_input_single_output()
+        model_lstm_single = network_architectures.lstm_network_multi_input_single_output()
```

Comment:
```suggestion
        model_mlp_multi_input_single_output = network_architectures.mlp_multi_input_single_output()
        model_lstm_multi_input_single_output = network_architectures.lstm_network_multi_input_single_output()
```

please try to name consistently